src/config.rs

```diff
@@ -257,3 +257,34 @@ impl<'config> TryFrom<&'config ArgMatches<'config>> for Config<'config> {
         Ok(builder.build())
     }
 }
+
+fn parse_version(input: &str) -> Result<semver::Version, semver::Error> {
+    match input {
+        "2018" => Ok(semver::Version::new(1, 31, 0)),
```

Comment:
For completeness sake, we should also add the first edition, i.e. "2015" (version 1.0.0). 

https://doc.rust-lang.org/edition-guide/rust-2015/index.html

Reply:
done!